metadata.json

```diff
@@ -5,6 +5,6 @@
   "gettext-domain": "AutomaticShutdownTimer",
   "settings-schema": "org.gnome.shell.extensions.automatic-shutdown-timer",
   "url": "https://github.com/mkrajnak/shutdown-timer-gnome-shell-extension/",
-  "shell-version": [ "3.32", "3.30", "3.28", "3.26", "3.24", "3.22", "3.20", "3.16", "3.14"],
+  "shell-version": ["40", "3.32", "3.30", "3.28", "3.26", "3.24", "3.22", "3.20", "3.16", "3.14"],
```

Comment:
Tested quickly with older machine that has GNOME3.32 and since it's without GTK4 it won't work with that. Personally, I am ok with that since user have an option to choose the shell version in extension website.

Please can you delete the versions prior to "40" from the list ? This way the users with older GNOME should download the older version.

Reply:
Hi, i just did that.

Do you want me to bump the version to someting like `0.10` as well?

Reply:
yes please !

Reply:
I will merge this and bumb the version in master so I can submit the zip for review. But again many thanks for this !